### DISCUSSION TITLE
Is it possible to specify multiple libraries using Pandoc export with the Pandoc filter?

### DISCUSSION THREAD

**Discussion by alexdaminger** [retorquere/zotero-better-bibtex#3024] [Q&A] [answered]
I am trying to convert an .md-file to .docx with Zotero live citations using the [Pandoc filter](https://retorque.re/zotero-better-bibtex/exporting/zotero.lua).

Everything goes smooth; however, I am trying to figure out whether it is possible to specify multiple libraries in the YAML-header as my citations are both, within my personal library and within a group library.

The [documentation](https://retorque.re/zotero-better-bibtex/exporting/pandoc/index.html) states that once the “library” argument is omitted, the personal library is used. Is there a way to specify both the personal library and a group library?

```
---
zotero:
  library: <group name> # omitted to use your personal library
---
```

**Comment by retorquere** [comment, marked as answer]
It isn't currently. I think it will be possible to add this, but you'll have to open an issue for changes.
